### PR TITLE
Set up unit testing with Jest. Write sample test code.

### PR DESCRIPTION
**Pull request by saraholiviawu**
**Description**: Set up code testing organization using Jest as a JavaScript test runner and assertion library. Will delete sample code. Jest is useful for mocking by default and supporting promises; Jest already includes an assertion library.

**Key notes for use**: 
  - Must export every JS function from the 'webapp' directory to be used in a file from the 'test' directory.
  - Each test file must include the syntax ".test.js" as a suffix to be recognized by Jest

**Future steps**:
  - Test DOM manipulation
  - Mock Firebase Firestore API
  - Mock localStorage
  - Write any other applicable unit tests

### CODE REVIEW

File: ripple/src/main/webapp/script.js [hunk 1]
```diff
@@ -129,19 +129,14 @@ function searchInput(keyPress) {
     var searchString = document.getElementById("search-bar").value;
     console.log(searchString);
     localStorage.setItem("searchString", searchString);
-    localStorage.setItem("searchStringRaw", )
+    localStorage.setItem("searchStringRaw", );
```
Comment (saraholiviawu) [inline ripple/src/main/webapp/script.js:132]: Ignore syntax error. Smruthi to update in next merge.

File: ripple/src/main/webapp/posts.js [hunk 2]
```diff
@@ -193,4 +193,13 @@ function createPostElement(userName, userBlobKey, businessName, postBlobKey, cap
   `;
   postElement.innerHTML = content;
   return postElement;
+}
+
+// Unit testing exports set up
+
+function subtract(a, b) {
```
Comment (sabadas) [inline ripple/src/main/webapp/posts.js:200]: is this just a test method that you will be writing unit tests for? (assuming will be deleted in a future PR)

File: ripple/src/main/webapp/posts.js [hunk 3]
```diff
@@ -193,4 +193,13 @@ function createPostElement(userName, userBlobKey, businessName, postBlobKey, cap
...
+  return a - b;
+}
+module.exports = {
```
Comment (sabadas) [inline ripple/src/main/webapp/posts.js:203]: is this also specifically for testing?

File: ripple/src/main/webapp/posts.js (hunk 2 above)
Comment (saraholiviawu) [reply ripple/src/main/webapp/posts.js:200]: yes, the function subtract is just a test method that will be deleted. It was written to confirm that the Jest has been properly set up

File: ripple/src/main/webapp/posts.js (hunk 3 above)
Comment (saraholiviawu) [reply ripple/src/main/webapp/posts.js:203]: Each function to be tested needs to be exported via module.exports

Comment (sabadas) [reply ripple/src/main/webapp/posts.js:203]: Ahh interesting, is that Jest specific or is it just how Javascript works?

File: ripple/src/main/webapp/script.js (hunk 1 above)
Comment (smruthibalajee) [reply ripple/src/main/webapp/script.js:132]: Should be resolved in the pull request I just merged!

File: ripple/src/main/webapp/posts.js (hunk 3 above)
Comment (saraholiviawu) [reply ripple/src/main/webapp/posts.js:203]: This is how JS works. I implemented this with Mocha / Chai earlier and had to export functions as well.

**Pull request merged**